gfx.cpp

```diff
@@ -1001,15 +1002,15 @@ class GfxInternal
 
     GfxResult initialize(uint32_t width, uint32_t height, GfxCreateContextFlags flags, IDXGIAdapter *adapter, GfxContext &context)
     {
-        IDXGIFactory4 *factory = nullptr;
+        IDXGIFactory1 *factory = nullptr;
         if(!SUCCEEDED(CreateDXGIFactory1(IID_PPV_ARGS(&factory))))
```

Comment:
Use IDXGIFactory1 directly instead of implicit cast. Not needed specifically but picks up errors on older windows platforms

Reply:
ha, good catch!